python/lsst/ts/Dome/mock_controller.py

```diff
@@ -2,7 +2,9 @@
 import logging
 import yaml
 
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.INFO
+)
```

Comment:
See comment above about calling basicConfig in library code. Having it here means it will be called twice, which is is not recommended.

Reply:
Thanks for the suggestion. You are right of course. I have removed the logging config from the library files and put them inside the unit test files and the Dome CSC start script.